share/lib/python/neuron/rxd/region.py

```diff
@@ -180,6 +180,8 @@ def _initalize(self):
         for rid, r in enumerate(self._regions):
             for sid, s in enumerate(self._react_species + self._react_params):
                 indices = s()._indices1d(r())
+                if len(indices) > self.num_segments:
```

Comment:
Do we currently support 1d subsegment discretization? I'm guessing we don't, but maybe leave a comment that this should be changed if we do in the future.